fido/src/main/java/com/yubico/yubikit/fido/ctap/Ctap2Session.java

```diff
@@ -694,7 +716,15 @@ private InfoData(
         int uvModality,
         Map<String, Object> certifications,
         @Nullable Integer remainingDiscoverableCredentials,
-        @Nullable List<Integer> vendorPrototypeConfigCommands) {
+        @Nullable List<Integer> vendorPrototypeConfigCommands,
+        @Nullable List<String> attestationFormats,
+        @Nullable Integer uvCountSinceLastPinEntry,
+        @Nullable Boolean longTouchForReset,
+        @Nullable byte[] encIdentifier,
+        @Nullable List<String> transportsForReset,
+        @Nullable Boolean pinComplexityPolicy,
+        @Nullable byte[] pinComplexityPolicyURL,
+        @Nullable Integer maxPINLength) {
```

Comment:
I *believe* we could default to 63 for maxPINLength, as prior to this field being available, that was the specified max length. Also, consider if it should be named "maxPinLength" for consistency with other properties, eg. "forcePinChange".